src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -174,6 +182,84 @@ private static async Task HandleSingleTypeAsync(CompletionContext context, Seman
                         filterText: memberDisplayName));
                 }
             }
+            else if (GetEnclosingNamedType() is { } enclosingNamedType)
+            {
+                // Build a list of the members with the same type as the target
+                foreach (var member in type.GetMembers())
+                {
+                    if (member is IFieldSymbol { IsStatic: true } field)
+                    {
+                        if (!SymbolEqualityComparer.Default.Equals(type, field.Type))
+                            continue;
+
+                        if (!field.IsAccessibleWithin(enclosingNamedType))
+                            continue;
+
+                        if (!field.IsEditorBrowsable(hideAdvancedMembers, semanticModel.Compilation))
+                            continue;
+
+                        AddTypeIfNotAdded();
+                        var memberDisplayName = $"{displayText}.{field.Name}";
+                        context.AddItem(SymbolCompletionItem.CreateWithSymbolId(
+                            displayText: memberDisplayName,
+                            displayTextSuffix: "",
+                            symbols: ImmutableArray.Create<ISymbol>(field),
+                            rules: CompletionItemRules.Default,
+                            contextPosition: position,
+                            sortText: memberDisplayName,
+                            filterText: memberDisplayName));
+                    }
+                    else if (member is IPropertySymbol { IsStatic: true, IsIndexer: false } property)
+                    {
+                        if (!SymbolEqualityComparer.Default.Equals(type, property.Type))
+                            continue;
+
+                        if (!property.IsAccessibleWithin(enclosingNamedType))
+                            continue;
+
+                        if (!property.IsEditorBrowsable(hideAdvancedMembers, semanticModel.Compilation))
+                            continue;
+
+                        AddTypeIfNotAdded();
+                        var memberDisplayName = $"{displayText}.{property.Name}";
+                        context.AddItem(SymbolCompletionItem.CreateWithSymbolId(
+                            displayText: memberDisplayName,
+                            displayTextSuffix: "",
+                            symbols: ImmutableArray.Create<ISymbol>(property),
+                            rules: CompletionItemRules.Default,
+                            contextPosition: position,
+                            sortText: memberDisplayName,
+                            filterText: memberDisplayName));
+                    }
+                    else
+                    {
+                        continue;
+                    }
```

Comment:
i'd just remove this.

Reply:
⏱️ Cleaned up as part of other work in progress